hapi-utils/src/main/java/com/hedera/services/legacy/proto/utils/SignatureVerifier.java

```diff
@@ -238,9 +239,12 @@ public static void main(String[] args)  {
     byte[] encKey = null;
     try (FileInputStream  keyfis = new FileInputStream(args[0])) {
         encKey = new byte[keyfis.available()];
-        keyfis.read(encKey);
+        int bytesRead = keyfis.read(encKey);
+        if (bytesRead != encKey.length) {
+          log.warn("Total bytes read from key file doesn't match expected length");
```

Comment:
Actually this code was repeated. We should extract it out to a helper function. And add a test for it.

Reply:
As SignatureVerifier. main is legacy code, and not used anywhere. This is removed.